Code/Tools/AssetProcessor/native/ui/MainWindow.ui

```diff
@@ -335,7 +335,7 @@
       <item>
        <widget class="QStackedWidget" name="dialogStack">
         <property name="currentIndex">
-         <number>0</number>
+         <number>4</number>
```

Comment:
Can you set this back to 0? I think this is the tab it defaults to opening to, we want the Asset Processor to start on the Jobs tab, not the Builder tab.

This is easy to miss, when you use Qt Designer to edit these files, it saves whatever tab you had active, which is usually what you're working on.

Reply:
yeah, we want that back to 0 to get the current behavior

Reply:
Thanks for pointing this out, and I will change it back to 0.
It seems that AP still starts from Jobs tab on my computer because MainWindow.cpp overwrites this config to avoid unintended errors like this.

Reply:
resolved